StylableSwiftUI/Classes/Stylist.swift

```diff
@@ -71,9 +71,15 @@ public class Stylist: ObservableObject {
 
     func style(view: Stylable, identifier: StylistIdentifier) -> some View {
 
-        // Apply the first matching style in our list of styles
-        guard let style = self.styles.first(where: { $0.identifier.matches(identifier) }) else {
-            print("No matching style found for ", identifier)
+        // Apply the most specific style we can find
+        let styles = self.styles
+            .filter { $0.identifier.matches(identifier) }
+            .sorted { a, b in
```

Comment:
_oh_. I wrote the code so that the array of styles was sorted at the point of adding, so applying was fast (you should only need to sort this collection once) so I'd like to spend some time looking at why that sort isn't working first :| In theory, you shouldn't need to sort every time.

Or, obviously, I've missed something obvious :)

Reply:
erm, I've just checked out you branch, reverted this change, and run the tests to see what's going wrong, but they pass :|

Can you try that on you machine please? Something _fishy_ is going on . . .

Reply:
FYI: Test passes no matter which order I apply the styles in the test.

Reply:
@deanWombourne if you run the example app the text at the bottom isn't red, but it should be?

Reply:
I obvs missed the sorting at insertion point

Reply:
OK, take a look at the commit I just added - fixes the example app and still passes the tests.